src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -54,38 +54,25 @@ private static Window InternalGetCurrentWindow()
 
 		internal void RaiseNativeSizeChanged(int screenWidth, int screenHeight)
 		{
-			int statusBarHeightDp = 0;
-
-			var activity = ContextHelper.Current as Activity;
-			var decorView = activity.Window.DecorView;
-			var isStatusBarVisible = ((int)decorView.SystemUiVisibility & (int)SystemUiFlags.Fullscreen) == 0;
-
-			if (isStatusBarVisible)
-			{
-				int resourceId = Android.Content.Res.Resources.System.GetIdentifier("status_bar_height", "dimen", "android");
-				if (resourceId > 0)
-				{
-					statusBarHeightDp = (int)(Android.Content.Res.Resources.System.GetDimensionPixelSize(resourceId) / Android.App.Application.Context.Resources.DisplayMetrics.Density);
-				}
-			}
-
 			var newBounds = ViewHelper.PhysicalToLogicalPixels(new Rect(0, 0, screenWidth, screenHeight));
 
 			if (Bounds != newBounds)
 			{
 				Bounds = newBounds;
 
+				var applicationView = ApplicationView.GetForCurrentView();
+				if (applicationView != null)
+				{
+					applicationView.SetCoreBounds(newBounds);
```

Comment:
This will only be called if Bounds changes. What if the VisibleBounds change without Bounds changing? Eg if the StatusBar is hidden/displayed?

Reply:
I've extracted it from newBounds and check for its own change